functions-methods.js

```diff
@@ -33,4 +82,35 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+//-----------------PLAN VAN AANPAK-----------------//
+//[] Checken of @ voorkomt in de input
+//[] Checken of , voorkomt in de input
+//[] Checken wat het laatste waarde is
+//[x] Als die laatste waarde overeenkomt met . dan false teruggeven
+//[x] Wanneer er true false en false wordt meegegeven dan is de waarde true, zo niet dan false
+
+console.log("Opdracht 3:");
+console.log("----------------------");
+
+function checkEmailValidity(email) {
+    const checkSymbol01 = email.includes("@");
+    const checkSymbol02 = email.includes(",");
+    const emailLength = email.length;
+    const lastIndex = email.substring(emailLength - 1);
+    if (checkSymbol01 === true && checkSymbol02 === false && lastIndex !== ".") {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Dit kun je nog korter schrijven 😉 
```suggestion
    return checkSymbol01 && !checkSymbol02 && lastIndex !== ".";
```